### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by outoftime** [popcodeorg/popcode#366]
Two difficult dependencies to update:
- [ ] Firebase: the new SDK no longer supports session-only login, and no longer holds on to the user’s GitHub token for them. Also, the code paths for Firebase login spread knowledge of Firebase’s auth response data structure too widely in the app, and it’s next to impossible to test at the moment
- [ ] I18next: I haven’t figured out how to do the whole “just load the translations from memory” thing in the new version of the client

**Comment by schraj** [comment]
For the i18next issue, I think this may be what you are looking for.  
https://github.com/i18next/i18next-xhr-backend

I just did this for this for another project(@5Calls).  Here is the i18next init file I'm using.
https://github.com/schraj/5calls/blob/i18n/static/js/utils/localization.js

**Comment by outoftime** [comment]
@schraj my bad for leaving this around and so out of date—I was able to upgrade to latest `i18next`, ended up using the Webpack loader to inline the translations. That approach definitely wouldn’t scale if we supported a significant number of languages, but for now it’s convenient not to have to wire up the UI to wait for the translations to load before drawing.

Gonna close this issue since both of the libraries in the description are up to date (or close enough).
